
# HG changeset patch
# User Timo Sirainen <tss@iki.fi>
# Date 1246987224 14400
# Node ID 8f8b5e2f8df7dc6e68630bc47df6ac0e27f344d2
# Parent d6337be8ae305ee5d095b806f10afc3207f3a261
pop3: Fixed AUTH PLAIN command when SASL initial response wasn't specified.

--- a/src/pop3-login/client-authenticate.c	Tue Jul 07 12:40:31 2009 -0400
+++ b/src/pop3-login/client-authenticate.c	Tue Jul 07 13:20:24 2009 -0400
@@ -319,7 +319,7 @@ bool cmd_auth(struct pop3_client *client
 	p = strchr(args, ' ');
 	if (p == NULL) {
 		mech_name = args;
-		args = "";
+		args = NULL;
 	} else {
 		mech_name = t_strdup_until(args, p);
 		args = p+1;

